torch_tvm/operators.cpp

```diff
@@ -183,27 +217,60 @@ RegisterTVMOperator reg({
        conv_attrs->groups = relayToConstant<int>(inputs[8]);
        conv_attrs->data_layout = "NCHW";
        conv_attrs->kernel_layout = "OIHW";
+       conv_attrs->strides = relayToArray<tvm::relay::IndexExpr>(inputs[3]);
+       conv_attrs->padding = relayToArray<tvm::relay::IndexExpr>(inputs[4]);
+       conv_attrs->dilation = relayToArray<tvm::relay::IndexExpr>(inputs[5]);
 
        conv_attrs->kernel_size =
            tvm::NullValue<tvm::Array<tvm::relay::IndexExpr>>();
+       // For 1D conv need to expand dim. That means it has to be removed
+       // as well.
+       bool dim_added{false}; // For 1D conv need to expand dim.
        // If the input was a complete tensor type than we have information to
        // populate the kernel
        if (const tvm::relay::VarNode* var =
                inputs[1].as<tvm::relay::VarNode>()) {
          auto* w_t = var->type_annotation.as<tvm::relay::TensorTypeNode>();
          TORCH_INTERNAL_ASSERT(w_t);
          auto shape = w_t->shape;
-         tvm::Array<tvm::relay::IndexExpr> w_sizes = {shape[2], shape[3]};
+         auto num_dims = shape.size();
+         tvm::Array<tvm::relay::IndexExpr> w_sizes;
+         if (num_dims < 4) {
+           AT_CHECK(num_dims == 3,
```

Comment:
I think we need to use the new CHECK api (TORCH_CHECK?)

Reply:
There is not TORCH_CHECK and the only other one is internal_assert.